components/org.wso2.carbon.identity.sso.agent/src/main/resources/META-INF/secretkey_temp.txt

```diff
@@ -0,0 +1 @@
+1234567812345678
```

Comment:
Why a default file ? Default should be plain text right ? So we don't need a file right ?